recipes/quill/all/conanfile.py

```diff
@@ -13,8 +13,14 @@ class QuillConan(ConanFile):
     exports_sources = ["CMakeLists.txt"]
     generators = "cmake", "cmake_find_package"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"fPIC": [True, False]}
-    default_options = {"fPIC": True}
+
+    options = {"fPIC": [True, False],
+               "with_bounded_queue": [True, False],
+               "with_exceptions": [True, False]}
+
+    default_options = {"fPIC": True,
+                       "with_bounded_queue": False,
+                       "with_exceptions": True}
```

Comment:
```suggestion
                       "with_exceptions": False}
```

Follow the same default value from the upstream https://github.com/odygrd/quill/blob/v1.3.2/CMakeLists.txt#L10

Reply:
I've changed it now so that the option has the same name and value as the cmakefile